### DISCUSSION TITLE
Window Error

### DISCUSSION THREAD

**Discussion by ghost** [dotnet/maui#10143] [Q&A]
**_error CS1061: 'Window' does not contain a definition for 'Height' and no accessible extens 
ion method 'Height' accepting a first argument of type 'Window' could be found (are you missing a using directive or an assembly reference?)_** 

I get this error when I use Window function to get width of the window. Please Help

Cheers.

**Comment by LucianoSponton** [comment]
Hi, Could you solved this? I have the same issue
